sdk/storage/azure-storage-files-datalake/inc/azure/storage/files/datalake/protocol/datalake_rest_client.hpp

```diff
@@ -760,7 +760,7 @@ namespace Azure { namespace Storage { namespace Files { namespace DataLake {
           return DeleteParseResult(context, pipeline.Send(context, request));
         }
 
-        struct ListDataLakePathsOptions
+        struct ListPathsOptions
```

Comment:
Is this exposed to end customers? If so, we need to add it to changelog

Reply:
It is not.